app/services/pagarme/response_parser.rb

```diff
@@ -0,0 +1,8 @@
+class Pagarme::ResponseParser
+  def self.process(json_objects)
+    formated_json = "[#{json_objects.gsub("\n\u0000", ',')[0..-2]}]"
+    return JSON.parse(formated_json).map do |subscription|
```

Comment:
Do you really need this `return`? Since it's the last evaluated expression it will be returned.